skipper.go

```diff
@@ -1294,6 +1296,49 @@ func getRedisUpdaterFunc(namespace, name string, kdc *kubernetes.Client) func()
 	}
 }
 
+type RedisEndpoint struct {
+	Url string `json:"url"`
+}
+
+type RedisEndpoints struct {
+	Endpoints []RedisEndpoint `json:"endpoints"`
+}
+
+func updateEndpointsFromURL(address string) func() []string {
+	return func() []string {
+		/* #nosec */
+		resp, err := http.Get(address)
+		if err != nil {
+			log.Errorf("failed to connect to redis endpoint %v, due to: %v", address, err)
+			return nil
+		}
+		defer resp.Body.Close()
+
+		body, err := io.ReadAll(resp.Body)
+		if err != nil {
+			log.Errorf("failed to read to redis response %v", err)
+			return nil
+		}
+
+		target := &RedisEndpoints{}
+
+		err = json.Unmarshal(body, target)
+		if err != nil {
+			log.Errorf("Failed to decode body to json %v", err)
+			return nil
```

Comment:
it doesn't reset `SetAddrs` function will ignore it len is 0 https://github.com/zalando/skipper/blob/693a042dcfdeb771930fa86eac2826f9d7ee9ae3/net/redisclient.go#L369-L374

Reply:
@AlexanderYastrebov that's only true in case you want to support scale down to 0, which is not really working anywhere. So actually it's fine to not change the interface.